pytorch_lightning/trainer/trainer.py

```diff
@@ -513,7 +513,6 @@ def run_sanity_check(self, ref_model):
             self.on_sanity_check_end()
             self.running_sanity_check = False
 
-    @trainer_state(entering=TrainerState.RUNNING, exiting=TrainerState.FINISHED)
```

Comment:
@Borda  let's please get rid of these... makes readability impossible (and IDE inspection)

Reply:
ok, we can remove wrappers, just keep the state assignment..